src/Generator.Rewrite/Program.cs

```diff
@@ -804,6 +842,72 @@ private static GeneratedVariableIdentifier EmitStringBuilderParameter(MethodDefi
             return generatedVariables;
         }
 
+        private class CountAttribute
```

Comment:
Yep, that's what meant :) Move to separate file and make internal. It's a weaker modifier, but strong enough for this use case.